### ISSUE TITLE
Running pipeline without sample groups

### ISSUE THREAD

**Issue by jgriss** [bigbio/proteomicstmt#28]
Hi,

@ypriverol as suggested I'm now testing the pipeline in more detail.

## Setup

I have a project where we characterised samples in a TMT experiment, but don't have clear sample groups yet (it's a phenotyping experiment).

## Problem

The pipeline runs fine until the `MsstatsConverter` (log output):
```
Debug level: 10
 >> MSstatsConverter -in ID_mapper_merge_epi_filt_resconf.consensusXML -in_design experimental_design.tsv -method ISO -out out_msstats.csv -debug 10
Value of string option 'test': 0
The OpenMS team is collecting usage statistics for quality control and funding purposes.
We will never give out your personal data, but you may disable this functionality by 
setting the environmental variable OPENMS_DISABLE_UPDATE_CHECK to ON.
Connecting to REST server failed. Skipping update check.
Error: Error transferring http://openms-update.informatik.uni-tuebingen.de/check/OpenMS_Default_Linux_64_MSstatsConverter_2.6.0-pre-develop-2021-06-08 - server replied: Internal Server Error
Debug level (after ini file): 10
Value of string option 'no_progress': 0
Value of string option 'in': ID_mapper_merge_epi_filt_resconf.consensusXML
Checking input file 'ID_mapper_merge_epi_filt_resconf.consensusXML'
Value of string option 'method': ISO
Value of string option 'out': out_msstats.csv
Checking output file 'out_msstats.csv'
Value of string option 'in_design': experimental_design.tsv
Checking input file 'experimental_design.tsv'
Error: Unexpected internal error (the value '1' was used but is not valid; Label not consecutive. Expected: 2)
Error occurred in line 705 of file /opt/conda/conda-bld/openms-meta_1623208805577/work/src/openms/source/METADATA/ExperimentalDesign.cpp (in function: void OpenMS::ExperimentalDesign::isValid_()) !
```

The (slightly anonymized) experimental_design.tsv is here (stored as .txt otherwise it's blocked by GitHub): [experimental_design.tsv.txt](https://github.com/bigbio/proteomicstmt/files/7264587/experimental_design.tsv.txt)

Looking at the `experimental_design.tsv` it seems that the `Fraction Group` is not recognized correctly. The experiment consists of 3 batches, each split across a number of fractions. The `Sample` and `Label` seem fine though.

## Expected Result

From this experiment, I naturally don't want / need any differential expression analysis.

What I'd expect is the **normalised protein-level abundance values** for each sample (similar to the output generated by FragPipe).

**Comment by daichengxin** [comment]
This experimental strategy is a bit special and may require manual normalization. Can you send me the raw data? I need to test it first. Thanks @jgriss

**Comment by daichengxin** [comment]
[merkel_auto_fixed.sdrf.txt](https://github.com/bigbio/proteomicstmt/files/7314738/merkel_auto_fixed.sdrf.txt)
This is the modified sdrf. Hope it will work

**Comment by daichengxin** [comment]
Yes. It's the identical sample. I discussed with @jpfeuffer and @timosachsenberg . The key problem is how to  describe that the same sample belongs to different mixtures in exp design file . But this is not supported in exp design file. One possible solution is to convert the same sample belonging to different mixtures in the sdrf into different samples in the experimental design file. I can try do it. If the bridge channel doesn't cross multiple mixtures, there is no such problem.

**Comment by jgriss** [comment]
Hi @daichengxin / @ypriverol 

First, to clarify a key point: My analysis is purely a benchmark of the workflow. I already analysed this dataset using two different strategies and already have my "final" results. @ypriverol asked me to compare these results as a test of the workflow.

Bridge channels are a standard approach in TMT experiments. They will be annotated (and are already annotated) in SDRF following the approach I used in my SDRF file. Therefore, if this is not properly supported, the workflow will be unusable for all larger isobarically labelled studies.

I don't understand your last point:

   > If the bridge channel doesn't cross multiple mixtures, there is no such problem.

A bridge channel is always used to normalise different mixtures. That's its only use. It solves the issue that many studies analyse more samples than available channels. Therefore, the samples need to be split in multiple batches / mixtures. To still be able to compare these different measurements, you need a bridge channel for the normalisation step.

Therefore, this issue needs to be solved in the workflow and not by hacking the SDRF file.

In other words, you need some kind of code that is able to rebuild the mixture composition (ie test in what kind of combinations, the different `source name`'s occur.

**Comment by jpfeuffer** [comment]
Just for clarification, "sample" in the experimental design is more like a unique identifier of an entry in the experimental design.
If all entries in the "sample section" of the design except the sample numbers/identifiers are the same, they are considered technical replicates of the same physical sample (which a Pool in a different mixture would be in my opinion).

This is how I see it. For further information I am tagging @timosachsenberg and we could ask on Msstats (whose format we were adapting)

**Comment by jpfeuffer** [comment]
So @daichengxin I think this should be solved in the converter, to always enumerate samples

**Comment by daichengxin** [comment]
Yes, I agree with your opinion. Thanks a lot

> So @daichengxin I think this should be solved in the converter, to always enumerate samples
